src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import Avatar from '../avatar';
+import TooltipMixin from '../../mixins/tooltip';
+import ApiMixin from '../../mixins/apiMixin';
+import GroupState from '../../mixins/groupState';
+
+import {t} from '../../locale';
+
+const SuggestedOwners = React.createClass({
+  propTypes: {
+    event: React.PropTypes.object,
+  },
+
+  mixins: [
+    ApiMixin,
+    GroupState,
+    TooltipMixin({
+      selector: '.tip',
+      container: 'body'
+    })
+  ],
+
+  getInitialState(){
+      return {owners: undefined};
+  },
+
+  componentDidMount(){
+    this.fetchData(this.props.event);
+  },
+
+  componentWillReceiveProps(nextProps){
+    if(this.props.event != nextProps.event){
+      this.fetchData(nextProps.event);
```

Comment:
it does work in this case but it's fair to be defensive. wish i had elvis operator or whatever